jbpm-wb-common/jbpm-wb-data-filters-client/src/main/java/org/jbpm/workbench/df/client/list/DataSetQueryHelper.java

```diff
@@ -58,21 +60,31 @@ public void setDataSetReadyEvent(final Event<DataSetReadyEvent> event) {
         this.event = event;
     }
 
+    public void resetLookupOperations() {
+        dataSetHandler.resetAllOperations();
+    }
+
+    public void enableMultipleColumnSorting(boolean enabled) {
+        allowMultipleColumnSorting = enabled;
+    }
+
     public void lookupDataSet(final Integer offset,
                               final DataSetReadyCallback callback) {
         try {
-            // Get the sort settings
-            if (lastOrderedColumn == null) {
-                String defaultSortColumn = currentTableSetting.getTableDefaultSortColumnId();
-                if (!StringUtils.isBlank(defaultSortColumn)) {
-                    lastOrderedColumn = defaultSortColumn;
-                    lastSortOrder = currentTableSetting.getTableDefaultSortOrder();
+            if (!allowMultipleColumnSorting) {
```

Comment:
If we simply not set the sorting order in the table settings this change and the _allowMultipleColumnSorting_ could be avoided right?